### PR TITLE
Add support for inserting entries with TTL in write bench tool

### PR DESCRIPTION
**Pull request by ahsanbarkati**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1425)

### CODE REVIEW

File: badger/cmd/write_bench.go [hunk 1]
```diff
@@ -112,7 +115,14 @@ func writeRandom(db *badger.DB, num uint64) error {
 	for i := uint64(1); i <= num; i++ {
 		key := make([]byte, keySz)
 		y.Check2(rand.Read(key))
-		if err := batch.Set(key, value); err != nil {
+
+		e := badger.NewEntry(key, value)
+
+		if withTTL {
+			e = badger.NewEntry(key, value).WithTTL(1 * time.Second)
```
Comment (NamanJain8) [inline badger/cmd/write_bench.go:122]: You can have e.WithTTL(1*time.Second) instead of creating NewEntry(). The rest looks good.

Comment (ahsanbarkati) [reply badger/cmd/write_bench.go]: Thanks. Done

File: badger/cmd/write_bench.go [hunk 2]
```diff
@@ -101,6 +102,8 @@ func init() {
 		"If true, it badger will detect the conflicts")
 	writeBenchCmd.Flags().BoolVar(&compression, "compression", false,
 		"If true, badger will use ZSTD mode")
+	writeBenchCmd.Flags().BoolVar(&withTTL, "entry-ttl", false,
```
Comment (jarifibrahim) [inline badger/cmd/write_bench.go:105]: change this to accept an integer which will be seconds. Set default to zero.

File: badger/cmd/write_bench.go [hunk 3]
```diff
@@ -112,7 +115,12 @@ func writeRandom(db *badger.DB, num uint64) error {
 	for i := uint64(1); i <= num; i++ {
 		key := make([]byte, keySz)
 		y.Check2(rand.Read(key))
-		if err := batch.Set(key, value); err != nil {
+
+		e := badger.NewEntry(key, value)
+		if withTTL {
```
Comment (jarifibrahim) [inline badger/cmd/write_bench.go:120]: If the value of the flag is zero, do not set TTL, otherwise use it.

**Pull request merged**